packages/dashboard/src/app/api/useStoryApi.js

```diff
@@ -36,6 +40,8 @@ import storyReducer, {
 import { ERRORS } from '../textContent';
 import { useConfig } from '../config';
 
+const DRAFT_STORAGE_KEY = `${SESSION_STORAGE_PREFIX.LOCAL_AUTOSAVE_PREFIX}_auto-draft`;
```

Comment:
Nit: perhaps add a comment pointing to `getSessionStorageKey()` in the editor and vice-versa, should the key format ever change.